analyse.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Surface errors in this script to CI, so they get noticed.
+# See e.g. http://redsymbol.net/articles/unofficial-bash-strict-mode/ for explanation.
+set -e -u -o pipefail
+
+# FIXME: error if this argument is not provided
+number=$1
+
+# Ask the user to download this page's text themselves.
+echo "You'd like to analysei PR $number: you have to perform the first step yourself"
+read -p "Visit github.com/leanprover-community/mathlib4/issues/$number; press enter to continue"
+read -p "Make sure there are no 'hidden items': expand them if necessary; press enter to continue"
+read -p "Select everything (e.g. using Ctrl-A), copy it (e.g. using Ctrl-C) and paste it into a plain text file. Save this file as pr-text-$number.txt. Press enter if you have done this."
+
+cat pr-$number-text.txt | rg "marked this pull request|label|commented" > interesting-pr-$number.txt
```

Comment:
For scripts run by many people, I prefer grep, since rg is not (yet) universally installed
```suggestion
cat pr-$number-text.txt | grep "marked this pull request|label|commented" > interesting-pr-$number.txt
```

Reply:
Good point. I was not really trying to make the script widely usable; for instance, I presume this script assumes linux/Unix systems...
Simply replacing `rg` by `grep` fails; grep syntax is different. Since this is just a prototype, let me leave it as-is for now --- or, if you know the correct grep syntax, feel free to suggest it here!